### PR TITLE
chore: Add benchamrks files for using as a demo

### PR DESCRIPTION
**Pull request by ebebbington**
Copied over from services before removing index service

### CODE REVIEW

File: benchmarks/README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+Searching took an avg of 0.00014s using Map.forEach().
+Searching took an avg of 0.00001s using IndexService.search().
+
+Performing search with 100,000 records(s) for 10s.
+Searching took an avg of 0.00136s using Map.forEach().
+Searching took an avg of 0.00001s using IndexService.search().
+
+Performing search with 1,000,000 records(s) for 10s.
+Searching took an avg of 0.01379s using Map.forEach().
+Searching took an avg of 0.00001s using IndexService.search().
```
Comment (saragee3) [inline benchmarks/README.md:42]: what's the time on _just_ the first hit, so without memoization? haven't looked into Moogle thoroughly, but what does moogle do that `Map.forEach` couldn't do with a simple memo wrapper? i guess what i'm saying is that maybe the tests would be more telling if it were more than a string search. Idk, what do you think?

Comment (crookse) [reply benchmarks/README.md:42]: agreed. we should show the output of a single search to fill the gap of "what's the time on the first search in moogle compared to the time on map.foreach call"

Comment (saragee3) [reply benchmarks/README.md:42]: not strong feelings though... the package seems like it's doing what it is intended for.

**Review by Guergeiro — APPROVED** [review]
Seems okay.

**Comment by ebebbington** [comment]
will split this out into another issue too

**Comment by Guergeiro** [comment]
> will split this out into another issue too

Okay mate.
@crookse maybe we don't need to revert and test previous versions of Drash. Maybe we just run current Drash against these benchmarks.

**Pull request merged**